packages/core/upload/admin/src/components/AssetCard/ImageAssetCard.js

```diff
@@ -16,13 +16,22 @@ import {
 } from '@strapi/design-system/Card';
 import { IconButton } from '@strapi/design-system/IconButton';
 import Pencil from '@strapi/icons/Pencil';
+import Trash from '@strapi/icons/Trash';
 import { useIntl } from 'react-intl';
 import { getTrad } from '../../utils';
 
 const Extension = styled.span`
   text-transform: uppercase;
 `;
 
+const CardActions = styled(CardAction)`
```

Comment:
if having multiple items in CardActions is a real need / documented in the DS, maybe we could think about integrating the design related to this need on the DS directly, instead of having to customize the component, WDYT? cc @HichamELBSI @maevalienard

Reply:
I don't have any opinion to be honest :D

Reply:
@HichamELBSI That's nice 🌻 . Thanks! Didn't know this was possible with `styled-components`.

Reply:
PR is there: https://github.com/strapi/design-system/pull/508 Let's move the discussion over there. Thanks everyone!

Reply:
Many thanks!!